package.json

```diff
@@ -2,6 +2,8 @@
   "name": "graphql-request",
   "version": "0.0.0-dripip",
   "main": "dist/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/index.d.ts",
```

Comment:
This shouldn't be needed.

Reply:
Both `main` and `module` could use the same type definitions if `types` key is provided.

Reply:
This relates to not enabled `declaration` true above, correct?

Reply:
Yes. and yes.

Reply:
@znck Ok let's try it this way.